tests/plugins/test_koji_import.py

```diff
@@ -176,11 +176,16 @@ def is_string_type(obj):
 
 
 class BuildInfo(object):
-    def __init__(self, value, valid=True):
-        if valid:
-            self.build = BuildResponse({'metadata': {'annotations': {'help_file': value}}})
-        else:
-            self.build = BuildResponse({'metadata': {'annotations': {}}})
+    def __init__(self, help_file=None, help_valid=True, media_types=None):
+        annotations = {}
+        if media_types is None:
```

Comment:
Yes. The pulp_plugin does not need to be run, so there needs to be a distinction between media_types is False (and no annotation is created) and media_types is [] (create a V1 image).

But looking over the code, that's an overly complicated way of expressing it and it's better to just drop both lines.